components/SearchSuggestions/SearchSuggestions.jsx

```diff
@@ -23,25 +23,32 @@ class SearchSuggestions extends Component {
   render() {
     const recentList = this.props.recentSearch
     const popularList = this.props.popularSearch
-
     const suggestionItem = (term, i) => {
       let labelDOM = term
       const searchTerm = this.props.searchTerm
+      let exactMatch = false
       if (searchTerm.length > 0) {
         const idx = term.toLowerCase().indexOf(searchTerm.toLowerCase())
         if (idx !== -1) {
+          // check if exact match
+          exactMatch = idx === 0 && term.length === searchTerm.length
```

Comment:
Would it ever be the case that `term.length === searchTerm.length`, but they aren't the same word?